_posts/2016-01-06-format-support.md

```diff
@@ -0,0 +1,115 @@
+about. For example, support for [3i Slidebook 6 files](http://www.openmicroscopy.org/site/support/bio-formats/formats/3i-slidebook.html) was only added to
+Bio-Formats last May when [3i](https://www.intelligent-imaging.com) committed
+to developing the reader themselves.
+Obviously, we are very grateful for this, but that doesn’t change the fact
+that we had already spent years working on various versions of this format
+(our initial single-series Slidebook reader was released back in 2006 and
+obviously the work to produce it started even further back than that). Nikon
+ND2 and Zeiss CZI are other examples of formats with a complex design that has
+made them very difficult for us to support.
+
+##We won’t deliver something that doesn’t do the job well enough
+
+One thing to understand about our work, strategy and commitment for supporting
+all file formats, especially formats used in production-scale facilities that
+use tech like whole slide imaging, is that we insist on delivering as close to
+complete support as possible. This is important given the size of community we
+support, the breadth of applications that use our software, and the need for
+utility and reliability in the software we deliver to the community.
+
+With .mrxs, it is very hard and expensive to meet this goal. To be specific:
+
+1. The design decisions of 3D HISTECH with respect to image pyramid layout are
+   at odds with what we can reasonably handle within the infrastructure
+   currently in place. Our analysis suggests we will have to re-calculate
+   several of the resolution levels, because of choices 3D HISTECH has made in
+   their tiling strategy.  This will create a substantial performance penalty
+   for anyone using Bio-Formats to read this format.
+2. The strategy for storing binary data on disk in 3D HISTECH .mrxs
+   brightfield differs substantially from fluorescence images stored in .mrxs.
+   They are essentially two different file formats, thus doubling the work
+   required.
+3. Based on recent data submissions and information from the community, 3D
+   HISTECH scanners default to JPEG-XR compression when acquiring fluorescence
+   data. Another doubling of work and complexity, as we would need to support
+   both compressed and uncompressed data, in brightfield and fluorescence.
+
+These points are specific to this format but similar issues occur with other
+proprietary formats. As a team, we are not comfortable with releasing a reader
+implementation that works on a limited set of file format variants, or
+requires time consuming and computationally expensive reprocessing and pyramid
+creation, just because of the implementation choices made by the format
+designers.
+
+##A philosophical point about our funding and resources
+
+The OME Consortium and the wider community has worked steadily since 2002,
+funded mostly by grants from non-profit charities and public funders, to build
+tools for the wider scientific community.
```

Comment:
Maybe the first use should be 'wider development community' to distinguish between the two? (Unless it can be expressed otherwise).